geonode_mapstore_client/static/mapstore/configs/localConfig.json

```diff
@@ -629,6 +629,14 @@
                             "disableIf": "{!state('selectedLayerPermissions').includes('download_resourcebase') || context.isDocumentExternalSource(state('gnResourceData'))}",
                             "type": "dropdown",
                             "items": [
+                                {
+                                    "type": "plugin",
+                                    "name": "DownloadResource",
+                                    "cfg": {
+                                        "downloadMsgId": "gnhome.dataset",
+                                        "allowedSources": ["LOCAL"]
+                                    }
```

Comment:
The ActionNavbar config should not include cfg, this configuration just tells to the menu which item to render. The cfg should be applied to the original plugin configuration